### PR TITLE
Show Verification's User as a link

### LINKED ISSUE #112: Show Verification's User as a link
Currently, the return was verification contains the `user_id`, `name` and `email_id`.

We just need to covert:
https://github.com/MLH-Fellowship/SocioMark/blob/39eb2f05a1a2da1ca6f6dfee388474da737c6a57/frontend/src/Components/Post/Post.js#L138-L147

Into a link `/user/{user_id}`

### PR DESCRIPTION
**Pull request by MyrrhDev**
When you click on the Verify button, the modal now links to the original author's profile
closes #112

### CODE REVIEW

File: frontend/src/Components/Post/Post.js [hunk 1]
```diff
@@ -170,11 +170,11 @@ export default function Post({ handleDeletePost, post_initializer }) {
...
-            "Original Author: " +
-              details.author_name +
-              " (" +
-              details.author_email +
-              ")"
+            <A href={`/user/${details.author_id}`}>
+              "Original Author:
+              {details.author_name}({details.author_email}
+              )"{" "}
+            </A>
```
Comment (aitikgupta) [inline frontend/src/Components/Post/Post.js]: Not sure about the exact tailwind property but this should at least make it look like a link when hovered:

```suggestion
            Original Author: 
            <A href={`/user/${details.author_id}`} className="hover: text-color-700">
              {details.author_name} ({details.author_email}
              ){" "}
            </A>
```

**Review by aitikgupta — CHANGES_REQUESTED** [review]
The logic is correct @MyrrhDev :tada: , just some css needed, as currently, it's not obvious that it's a link. (And some formatting issues)

@bodhisha can you verify the correct CSS prop?

File: frontend/src/Components/Post/Post.js [hunk 2]
```diff
@@ -306,7 +306,7 @@ export default function Post({ handleDeletePost, post_initializer }) {
                   }}
                   show={showModal}
                 >
-                  {modalText}
+                  <A href={`/user/${post.user_id}`}> {modalText}</A>
```
Comment (bodhisha) [inline frontend/src/Components/Post/Post.js]: The A tag here is unnecessary.. A simple `{modalText}` would do the work.
```suggestion
                   {modalText}
```

**Review by bodhisha — CHANGES_REQUESTED** [review]

File: frontend/src/Components/Post/Post.js (hunk 1 above)
Comment (bodhisha) [reply frontend/src/Components/Post/Post.js]: `className="hover:text-blue-700"` would work

**Review by bodhisha — APPROVED** [review]
Awesome.. 🎉

**Comment by bodhisha** [comment]
Looks good to merge! 🎉

File: frontend/src/Components/Post/Post.js [hunk 3]
```diff
@@ -170,11 +170,13 @@ export default function Post({ handleDeletePost, post_initializer }) {
...
-              " (" +
-              details.author_email +
-              ")"
+            <A
+              href={`/user/${details.author_id}`}
+              //className="hover:text-blue-700"
+            >
+              "Original Author: <u>{details.author_name}</u> ({details.author_email}
+              )"{" "}
+            </A>
```
Comment (aitikgupta) [inline frontend/src/Components/Post/Post.js]:
```suggestion
            <A href={`/user/${details.author_id}`}>
              Original Author:{" "}
              <span className="hover:text-blue-700">
                {details.author_name} ({details.author_email}){" "}
              </span>
```

**Review by aitikgupta — APPROVED** [review]
Thankyou! Just commit my suggestion directly (prettier and styling fix) and it's good to merge! 🎉

EDIT: I can commit it myself, nevermind. :p

File: frontend/src/Components/Post/Post.js [hunk 4]
```diff
@@ -170,11 +170,11 @@ export default function Post({ handleDeletePost, post_initializer }) {
...
-            "Original Author: " +
-              details.author_name +
-              " (" +
-              details.author_email +
-              ")"
+            <A href={`/user/${details.author_id}`}>
+              Original Author:{" "}
+              <span className="hover:text-blue-700">
+                {details.author_name} ({details.author_email}){" "}
+              </span>
```
Comment (aitikgupta) [inline frontend/src/Components/Post/Post.js]:
```suggestion
            <A href={`/user/${details.author_id}`}>
              Original Author: &ensp;
              <span className="hover:text-blue-700">
                {details.author_name} ({details.author_email})
              </span>
            </A>
```

Comment (aitikgupta) [reply frontend/src/Components/Post/Post.js]: I missed an `</A>` when suggesting xD

**Pull request merged**